articles/files-response/snippets/separating-fixtures-from-test/readme.md

```diff
@@ -0,0 +1,7 @@
+# 01-bdd-integration-spec
+
+Czytając testy, ciężko odnaleźć esencję, co sprawdzamy i jakie są warunki wejściowe? Projekt nie jest gotowy na używanie notacji Cucumbera/Gherkina? Nie szkodzi, nadal możesz zorganizować testy w czytelny sposób!
+
+# 02-bdd-integration-spec-fixtures
+
+Zorganizowanie kodu potrzebnego do przeprowadzenia testu zwiększa czytelność zarówno samego testu, jak i "świata" tworzonego na jego potrzeby. 
```

Comment:
it's not only for organizing tests, just with a test-driven approach it's natural to organize it like this, otherwise it's hard to write tests first